cmake/install.cmake

```diff
@@ -50,18 +57,16 @@ LIST(FIND VALID_INSTALL_LAYOUTS "${INSTALL_LAYOUT}" layout_no)
 IF(layout_no EQUAL -1)
   MESSAGE(FATAL_ERROR "Invalid installation layout ${INSTALL_LAYOUT}. Please specify one of the following layouts: ${VALID_INSTALL_LAYOUTS}")
 ENDIF()
-
-
-
 #
 # Todo: We don't generate man pages yet, will fix it
 #       later (webhelp to man transformation)
 #
-
+IF(NOT INSTALL_LAYOUT STREQUAL "DEFAULT")
+  SET(CMAKE_INSTALL_PREFIX "/usr")
+ENDIF()
```

Comment:
You set `CMAKE_INSTALL_PREFIX` here, but you copy it into `libmariadb_prefix` 20 lines above. How comes?

Reply:
ok thanks